services/ui-src/src/additional-scss/_footer.scss

```
@@ -1,93 +1,183 @@
   FOOTER
 **********************/
-.footer {
-  background: $brand-tertiary;
-  display: flex;
-  flex-direction: column;
-  z-index: 2;
-
-  .container {
-    width: 100%;
-    padding: 2rem;
-  }
-
-  .logo {
-    margin-bottom: 1.2rem;
-
-    ul {
-      padding: 0;
-
-      li {
-        display: inline-block;
-        list-style-type: none;
-        margin-left: 0;
-        padding-right: 0.4rem;
-
-        img {
-          max-height: 50px;
-        }
-
-        &:last-of-type {
-          img {
-            max-height: 80px;
-          }
-        }
-      }
+.footerRoot {
+  flex-shrink: 0;
+  @media (min-width: 55.0625em)  {
+      position: sticky;
+      z-index: 1000;
+    }
+  @media print {
+    display: none;
+  }
+  .footerTop {
+    background: $brand-tertiary;
+    min-height: 7rem;
+    padding: 2rem 0;
+  }
+  .footerTopContainer {
+    max-width: 100vw;
+    @media (min-width: 55.0625em)  {
+      padding: 0 2rem;
+    }
+  }
+  .footerTopFlex {
+    display: flex;
+    flex-wrap: wrap;
+  }
+  .footerTopLeftContainer {
+    flex: 0 0 50%;
+    max-width: 100%;
+    min-width: 100%;
+    margin-bottom: 1em;
+      @media (min-width: 55.0625em)  {
+      margin-bottom: 0;
+      max-width: 50%;
+      min-width: auto;
+    }
+  }
+  .footerTopRightContainer {
+    flex: 0 0 50%;
+    max-width: 100%;
+    min-width: 100%;
+     @media (min-width: 55.0625em) {
+      min-width: auto;
+    }
+  }
+  .footerTopRightTopFlex {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    align-items: center;
+     @media (min-width: 55.0625em) {
+      justify-content: flex-end;
+      flex-direction: row;
+    }
+  }
+  .footerCMSBrandingLeft {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    max-width: 100%;
+    padding: 0 1rem;
+    min-width: 100%;
+    margin-bottom: 1rem;
+     @media (min-width: 55.0625em) {
+      margin-bottom: 0;
+      display: block;
+      width: auto;
+      max-width: auto;
+      min-width: auto;
+    }
+  }
+  .footerCMSBrandingRight {
+    flex: 0 0 75%;
+    max-width: 25rem;
+  }
+  .sedsLogo {
+    max-height: 4.875rem;
+    margin: 0 auto;
+    display: block;
+     @media (min-width: 55.0625em) {
+      margin: 0;
+    }
+  }
+  .hhsLogo { 
+    display: inline-block;
+    width: 4.375rem;
+  }
+  .hhsMedicaidLogoMobile {
+    display: block;
+    max-width: 12.5rem;
+    margin-left: 1.25rem;
+    // desktop view
+     @media (min-width: 55.0625em) {
+      display: none;
+    }
+  }
+  .hhsCopyText {
+    display: inline-block;
+    text-align: left;
+    vertical-align: top;
+    font-size: 0.875rem; // 14px
+    line-height: 21px;
+    margin-top: 0;
+    margin-block: 0;
+  }
+  .footerCMSMedicaid {
+    display: none;
+    justify-content: flex-end;
+    // desktop view
+     @media (min-width: 55.0625em) {
+      display: flex;
+      max-width: 100%;
+      padding: 0 1rem;
+      min-width: 100%;
+      margin-bottom: 1rem;
```

Only `display: flex;` is needed:

https://github.com/user-attachments/assets/0a7a4624-2451-4821-8780-465618dd86a6


```suggestion
      display: flex;
```